src/components/MeasurementOverlay/MeasurementOverlay.js

```diff
@@ -36,7 +33,14 @@ class MeasurementOverlay extends React.PureComponent {
     super(props);
     this.state = {
       annotation: null,
+      transparentBackground: false,
```

Comment:
@bollain Let me know if the usage of this state and `this.isCreatingAnnotation` is clear and easy to understand. If not I can write some comments in the code to explain.